### PR TITLE
Adds a basic "/lua" command through @robg204's gist

### PR DESCRIPTION
**Pull request by LiquidityC**

### CODE REVIEW

File: src/lua/lua_script.rs [hunk 1]
```diff
@@ -38,13 +38,17 @@ fn create_default_lua_state(writer: Sender<Event>, dimensions: (u16, u16)) -> Lu
...
-            ctx.load(bindings).exec()?;
-            let defaults = include_str!("../../resources/lua/defaults.lua");
-            ctx.load(defaults).exec()?;
+
+            ctx.load(include_str!("../../resources/lua/bindings.lua"))
+                .exec()?;
+            ctx.load(include_str!("../../resources/lua/defaults.lua"))
+                .exec()?;
+            ctx.load(include_str!("../../resources/lua/lua_command.lua"))
+                .exec()?;
```
Comment (Xandaros) [inline src/lua/lua_script.rs:51]: Should probably make this more general and just load all `resources/lua/*.lua`

Comment (LiquidityC) [reply src/lua/lua_script.rs:51]: Yup, but that doesn't work with `include_str!()` since that's done at compile. Unless you know something I don't :smile:

Comment (Xandaros) [reply src/lua/lua_script.rs:51]: Yeah, it's not trivial. You'll probably have to make a custom macro for it.
If you prefer, I can do that, too. Something like `lua_load_directory!(ctx: _, path: &str) -> Result<()>` (Don't know the type of context)

Comment (LiquidityC) [reply src/lua/lua_script.rs:51]: You are most welcome to give it a shot. This will have to do for the time being.

**Pull request merged**